construction/demo/account_analytic_account.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
```

Comment:
Please bring back "Internal" one + related project.task and account.analytic.line